M2/include/M2/config.h.cmake

```diff
@@ -57,14 +57,14 @@
 
 // TODO: only used in Macaulay2/d/interrupts.d. Still needed?
 /* Defined if you have the `alarm' function. */
-#cmakedefine HAVE_ALARM
+#cmakedefine HAVE_ALARM 1
```

Comment:
Just to make sure, do you want these to be `#define blah 1` if defined and `/* #undef blah */` otherwise, or do you want `#define blah 0` if it's not defined? In that case you want `#cmakedefine01 blah` here. See [here](https://cmake.org/cmake/help/latest/command/configure_file.html#transformations).

Reply:
They should be undefined, not 0, since they're used in `#ifdef`'s and `#if defined(...)`'s.